Tests/XcodeProjTests/Objects/Targets/PBXAggregateTargetTests.swift

```diff
@@ -27,4 +27,34 @@ final class PBXAggregateTargetTests: XCTestCase {
             "name": "name",
         ]
     }
+
+    func test_addDependency() throws {
+        let objects = PBXObjects(objects: [])
+
+        let configurationList = XCConfigurationList.fixture()
+        let mainGroup = PBXGroup.fixture()
+        objects.add(object: configurationList)
+        objects.add(object: mainGroup)
+
+        let project = PBXProject(name: "Project",
+                                 buildConfigurationList: configurationList,
+                                 compatibilityVersion: "0",
+                                 mainGroup: mainGroup)
+
+        objects.add(object: project)
+        let target = PBXNativeTarget(name: "Target", buildConfigurationList: nil)
```

Comment:
Should the test not create an `PBXAggregateTarget` seeing that's a the subject under test? I appreciate this may have been copied from the existing native target test 😅

Reply:
Ahh, yep.